src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -135,17 +129,18 @@ public function alter_context( Context $context, array $arguments = [] ) {
 	/**
 	 * Translates widget arguments to their Context argument counterpart.
 	 *
-	 * @todo update in TEC-3620 & TEC-3597
-	 *
-	 * @since 5.2.1
+	 * @since  5.2.1
 	 *
 	 * @param array<string,mixed> $arguments Current set of arguments.
 	 * @param Context             $context   The request context.
 	 *
 	 * @return array<string,mixed> The translated widget arguments.
 	 */
 	protected function args_to_context( array $arguments, Context $context ) {
-		$context_args = [ 'widget' => true ];
+		$context_args = [
+			'widget' => true,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 7 space(s) between "'widget'" and double arrow, but found 1.

<sub>hash: 3a190549fd0e6d06198f98a142cf973f</sub>